src/resolvers/UserResolver.js

```diff
@@ -0,0 +1,9 @@
+const User = require('../models/User');
+
+const root = {
+  getUser(username) {
```

Comment:
I would recommend calling this `user` rather than `getUser` as it is representing a field that the client will be hitting for a given entity.